src/bitmap/store.rs

```diff
@@ -43,6 +43,20 @@ impl Store {
         }
     }
 
+    pub fn push(&mut self, index: u16) {
+        match *self {
+            Array(ref mut vec) => {
+                if vec.is_empty() || vec.last().unwrap() < &index{
```

Comment:
```suggestion
                if vec.last().map_or(true, |x| x < &index) {
```

Maybe this could work and avoid an unwrap.

Reply:
looks good to me!